crates/protocol/interop/src/depset.rs

```diff
@@ -0,0 +1,225 @@
+use crate::MESSAGE_EXPIRY_WINDOW;
+use alloy_primitives::ChainId;
+use kona_registry::HashMap;
+
+/// Configuration for a dependency of a chain
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "serde", serde(rename_all = "camelCase"))]
+pub struct ChainDependency {
+    /// chain_index is the unique short identifier for this chain
```

Comment:
```suggestion
    /// Chain_index is the unique short identifier for this chain
```
for docs we use proper sentences (or skipping trailing punctuation if you like), for comments not